LocustScripts/project-config/bha-referrals-perf/config.yaml

```diff
@@ -0,0 +1,7 @@
+host: https://www.commcarehq.org
+domain: co-carecoordination-perf
+## Model Iteration
+app_id: f83dc516d1e3f382528198375de23adb
+domain_user_credential: project-config/bha-referrals-perf/user_credentials.json
+owner_id: 3ca0fb9a3402487590f9b2eb5f0c876e
+app_config: project-config/bha-referrals-perf/app_config.json
```

Comment:
similar to above, seems like some of these would be ideal for command line arguments

Reply:
[cc98536](https://github.com/dimagi/dimagi-qa/pull/323/commits/cc98536d4f5652ee2d5c370a90c955f67f0d2b60)